JSDemos/Demos/VectorMap/DynamicViewport/description.md

```diff
@@ -1 +1,5 @@
-This demo illustrates how to&nbsp;change the map&rsquo;s viewport from code. Choose a&nbsp;continent using the drop-down menu under the map. In&nbsp;response to&nbsp;that, the map changes the visible area so&nbsp;that the chosen continent is&nbsp;displayed optimally. This is&nbsp;performed in&nbsp;code by&nbsp;calling the **viewport(viewportCoordinates)** method. After that, callback functions assigned to&nbsp;the **centerChanged** and **zoomFactorChanged** properties are triggered. These functions place the current values of&nbsp;the map&rsquo;s center and zoom factor in&nbsp;the text boxes under the map.
\ No newline at end of file
+This demo illustrates how to change the map viewport. Use the drop-down menu under the map to choose a continent. In response to that, the map changes the visible area so that the chosen continent is displayed optimally. 
+
+To implement this functionality, call the [viewport(viewportCoordinates)](/Documentation/ApiReference/UI_Components/dxVectorMap/Methods/#viewportviewportCoordinates) method every time the drop-down box value changes. You can also implement the [onCenterChanged](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/#onCenterChanged) and [onZoomFactorChanged](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/#onZoomFactorChanged) functions. These functions place the current values of the map’s center and zoom factor in the text boxes under the map.
```

Comment:
Why do we need to implement the mentioned functions? To display the current values in text boxes under the map. If you try to make your sentences very short, it may lead to awkward repetitions...

```suggestion
To implement this functionality, call the [viewport(viewportCoordinates)](/Documentation/ApiReference/UI_Components/dxVectorMap/Methods/#viewportviewportCoordinates) method every time the drop-down box value changes. You can also implement the [onCenterChanged](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/#onCenterChanged) and [onZoomFactorChanged](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/#onZoomFactorChanged) functions to display the map center's coordinates and the current zoom factor in text boxes under the map.
```